src/lib/theme/_overrides.scss

```diff
@@ -188,6 +188,8 @@ $shape-radius: 8px;
     }
   }
   body {
+    color: if($is-dark, map-get($mat-dark-theme-foreground, text), map-get($mat-light-theme-foreground, text));
```

Comment:
All this if is-dark business is redundant, the SASS maps swap already when you change the theme. This does the same thing:

`color: map-get($background, text);`